README.md

```diff
@@ -504,12 +504,18 @@ Install HLS along with the following emacs packages:
 
 Make sure to follow the instructions in the README of each of these packages.
 
-Install [use-package](https://github.com/jwiegley/use-package), and add the following to your .emacs
+The default `lsp-haskell-server-path` is set to `haskell-language-server-wrapper`. In 
+case you would like your editor to use a specific version of the `hls` server, then this
+variable can be updated.
+
+Install [use-package](https://github.com/jwiegley/use-package), and add the following to your .emacs 
+when you want to customize this variable.
+
 ``` emacs-lisp
 (use-package lsp-haskell
  :ensure t
  :config
- (setq lsp-haskell-process-path-hie "haskell-language-server-wrapper")
+     (setq lsp-haskell-server-path "haskell-language-server-8.10.2")
```

Comment:
I think this should be commented with an appropriate explanation, as it's normally unneeded.

Reply:
I was also wondering if we even need this section. Should we entirely get rid of this code snippet? Setting, up the emacs variable could easily be done with `(setq lsp-haskell-server-path "haskell-language-server-8.10.2")` for example.  Therefore, we could just omit this code snippet.

Do you agree?

Reply:
Can we just link to the documentation in `lsp-haskell`? That's more likely to be (and remain) up-to-date.

Reply:
I think I made the relevant changes. Please review!